### PR TITLE
Clockwork proselytizers can proselytize tables

### PR DESCRIPTION
**Pull request by ChangelingRain**
aaaaaaaaaaaaAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAaaaaaaaaaaaaaaAAAAAAAAAAAAaaaaaaaaaaa

### CODE REVIEW

**Comment by AdamElTablawy** [comment]
STOP MAKING FEATURE PRS DURING A FREEZE YOU CUCK

**Comment by ChangelingRain** [comment]
no

**Comment by coiax** [comment]
Do you get any CV from brass tables

**Comment by ChangelingRain** [comment]
yes

**Comment by Cyberboss** [comment]
The freeze is a fucking meme

File: code/_onclick/item_attack.dm [hunk 1]
```diff
@@ -3,6 +3,9 @@
 /obj/item/proc/attack_self(mob/user)
 	return
 
+/obj/item/proc/pre_attackby(obj/O, mob/living/user, params) //do stuff before attackby!
+	return TRUE //return FALSE to avoid calling attackby after this proc does stuff
```
Comment (ChangelingRain) [inline code/_onclick/item_attack.dm]: uh yeah? returning FALSE would prevent any attackby from working, as the comment says right here

Comment (PKPenguin321) [inline code/_onclick/item_attack.dm]: Oh oh oh I misunderstood the comment

**Comment by KingPhilipIII** [comment]
MUH CLOCKWORK COMPUTERS WHEN.

**Pull request merged**